airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -92,7 +92,7 @@ def transform_bigquery(self, config: dict):
         dbt_config["type"] = "bigquery"
         dbt_config["method"] = "service-account"
         dbt_config["project"] = config["project_id"]
-        dbt_config["dataset"] = config["dataset_id"]
+        dbt_config["dataset"] = ""
```

Comment:
why are these changing?

Reply:
The destination config doesn't ask for dataset or schema anymore so you can't configure the DBT project to a default one.

But DBT requires the presence of a dataset anyway (which in this case is empty string "").
It could be set as a default random value instead though? "_airbyte" maybe?

But in any case, during normalization, DBT wouldn't write anything in that default schema because every final table will have its own specific namespace to write to (specified in the catalog)